libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -407,52 +404,45 @@ bool IRGeneratorForStatements::visit(Assignment const& _assignment)
 		assignmentOperator :
 		TokenTraits::AssignmentToBinaryOp(assignmentOperator);
 
-	Type const* rightIntermediateType =
-		TokenTraits::isShiftOp(binaryOperator) ?
-		type(_assignment.rightHandSide()).mobileType() :
-		type(_assignment.rightHandSide()).closestTemporaryType(
```

Comment:
For the record, I still need to verify the types and conversions here :). So nothing `closestTemporaryType` does is required here and above?

Reply:
I spent a lot of time to check these conversions, they look good to me.
So, nothing is required from `closestTemporaryType` both here and above.

But, doesn't hurt to get it double checked :)